plat/kvm/efi.c

```diff
@@ -319,6 +328,7 @@ static void uk_efi_setup_bootinfo_mrds(struct ukplat_bootinfo *bi)
 	__u32 rt_mrds_count = 0, i;
 	uk_efi_status_t status;
 	int rc;
+	uk_efi_uintn_t uk_efi_map_key;
```

Comment:
Let's try to keep a reverse xmas tree variable declaration :pray:. Since it no longer is a key global variable, let's also remove the `uk_efi_` prefix.